test/mitmproxy/contentviews/test_image_parser.py

```diff
@@ -0,0 +1,14 @@
+import io
+
+from mitmproxy.contentviews.image import image_parser
+from mitmproxy.test import tutils
+
+def test_png_parser():
```

Comment:
This definitely needs more tests. See http://www.schaik.com/pngsuite/#ancillary for test files. We should have those at least:
```
ct0 - no textual data
ct1 - with textual data
ctz - with compressed textual data
cte - UTF-8 international text - english
```